testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -1282,6 +1334,20 @@ impl RelayerThread {
         };
         let stacks_tip = StacksBlockId::new(&stacks_tip_ch, &stacks_tip_bh);
 
+        // see if we have to try and continue a tenure
+        if let Ok(ongoing_tenure_id) =
+            BlockMinerThread::get_ongoing_tenure_id(&self.sortdb, &mut self.chainstate).map_err(
+                |e| {
+                    error!("Failed to get ongoing tenure ID: {:?}", &e);
+                    e
+                },
+            )
+        {
+            if ongoing_tenure_id.burn_view_consensus_hash != sort_tip.consensus_hash {
+                todo!();
```

Comment:
Do we have an issue for this? D: If we plan to leave a todo! in the code, would be good to add a message to it.

Reply:
This has since been implemented.